Grievance_Backend/grievance/src/main/java/com/grievance/controller/DepartmentController.java

```diff
@@ -57,14 +58,18 @@ public class DepartmentController {
 //  }
   @PostMapping("/save")
   public ResponseEntity<?> saveDepartment(
-    @RequestParam final String email,
-    @RequestParam final String password,
+    @RequestHeader final String email,
+    @RequestHeader final String password,
```

Comment:
why we are passing email and password in header instead of Requestparm or RequestBody?